esp-hal/src/soc/esp32s3/psram.rs

```diff
@@ -12,6 +12,52 @@
 //!
 //! The mapped start address for PSRAM depends on the amount of mapped flash
 //! memory.
+//!
+//! ## Examples
+//!
+//! ### Octal/Quad PSRAM (depends on which feature of `octal-psram` or `quad-psram` is enabled)
```

Comment:
```suggestion
//! ### Octal/Quad PSRAM
```

This should probably be a paragraph below, doesn't need to be part of the title IMO.